Framework/DataHandling/src/LoadCalFile.cpp

```diff
@@ -98,13 +99,11 @@ Geometry::Instrument_const_sptr LoadCalFile::getInstrument3Ways(Algorithm *alg)
   if (inWS) {
     inst = inWS->getInstrument();
   } else {
-    Algorithm_sptr childAlg = alg->createChildAlgorithm("LoadInstrument", 0.0, 0.2);
-    MatrixWorkspace_sptr tempWS = std::make_shared<Workspace2D>();
-    childAlg->setProperty<MatrixWorkspace_sptr>("Workspace", tempWS);
+    Algorithm_sptr childAlg = alg->createChildAlgorithm("LoadEmptyInstrument", 0.0, 0.2);
```

Comment:
Tests of this branch of behavior occur in `LoadEmptyInstrumentTest.h`, which validates this child algorithm can load an instrument from valid nexus files.